ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14-periodics.yaml

```diff
@@ -40184,6 +40184,200 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 53 16 1,3,5,7,9,11,13,15,17,19,21,23,25,27,29 * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: release-4.14
+    org: openshift
+    repo: openshift-tests-private
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-sd-qe
+    ci-operator.openshift.io/variant: amd64-stable
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-stable-aws-rosa-sts-hypershift-sec-guest-prod-private-link-full-f2
+  reporter_config:
+    slack:
+      channel: '#ocm-qe-prow-ci-jobs'
```

Comment:
Not sure whether you still manage the HCP notification at `managed-hypershift-ci-watcher`, or in the future, in `ocm-qe-prow-ci-jobs`

Reply:
Currently the notification channel of all of the rosa hosted cluster jobs is `managed-hypershift-ci-watcher`
We could update them together if needed in the future. Thanks!

Reply:
Oh.. I need to change it to `managed-hypershift-ci-watcher`